client/components/product-card/action.jsx

```diff
@@ -22,6 +28,14 @@ ProductCardAction.propTypes = {
 	intro: PropTypes.oneOfType( [ PropTypes.string, PropTypes.node, PropTypes.element ] ),
 	label: PropTypes.string.isRequired,
 	onClick: PropTypes.func,
+	href: PropTypes.string,
+	isPrimary: PropTypes.boolean,
```

Comment:
In addition to renaming, `boolean` isn't a valid PropType, we should use `bool` instead.

```suggestion
	primary: PropTypes.bool,
```